### PR TITLE
Added date/time picker examples

### PR DESCRIPTION
**Pull request by rolandoldengarm**
Added both the wide version as well as the smaller version to show it's responsive.

### CODE REVIEW

File: ghdocs/COMPONENTS.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ For simple instructions for using Fabric components in Office Add-ins, see the a
 |Search box|![Search box](http://odux.azurewebsites.net/github/img/Search.png)|
 |Toggle|![Toggle](http://odux.azurewebsites.net/github/img/Toggle.png)|
 |Text fields|![Text field](http://odux.azurewebsites.net/github/img/TextFields.png)|
+|Date/time Picker|![Date/time picker](http://i.imgur.com/odFYZjM.png)![Date/time picker small](http://i.imgur.com/k0Yz8Bo.png)
```
Comment (wdo3650) [inline ghdocs/COMPONENTS.md]: Please change the component name to "Date Picker" and place it alphabetically in the list.

**Comment by wdo3650** [comment]
Thanks for adding these samples! After reviewing your PR, I've noted a few things.

**Comment by rolandoldengarm** [comment]
@wdo3650 Thank you, I've updated the PR!

**Comment by wdo3650** [comment]
Nice work @rolandoldengarm! This is approved.

**Pull request merged**